pyccel/stdlib/numpy/numpy_f90.f90

```diff
@@ -0,0 +1,86 @@
+! --------------------------------------------------------------------------------------- !
+! This file is part of Pyccel which is released under MIT License. See the LICENSE file   !
+! or go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details. !
+! --------------------------------------------------------------------------------------- !
+
+module numpy_f90
+
+    use ISO_C_BINDING
+    
+    implicit none
+    
```

Comment:
You should add private and public declarations to ensure that if you do `use numpy_f90` only `numpy_sign` is imported.

I.e.
```
module numpy_f90

    use ISO_C_BINDING

    implicit none

    private

    public :: numpy_sign


    interface numpy_sign
    ...
```

Reply:
https://github.com/pyccel/pyccel/pull/1169/commits/f882cc173f3165b9bc46a35f4aec0fc3e411dee7